docs/accessibility.md

```diff
@@ -0,0 +1,39 @@
+# Accessibility Testing using Cypres and Axe
+
+This codebase comes with accessibility (a11y) testing in the form of [Axe](https://www.deque.com/axe/) combined with [Cypress](https://www.cypress.io/).
+
+In order to set up accessibility testing locally, clone and install the project as per the instructions, including making sure the Docker image is fully set up, and then run the Cypress/Axe installs using:
+
+```
+npm run cypress:install
+```
+
+Note that this should be run locally, _outside of Docker_. Also note that this will take quite a while to install, so be prepared to make some coffee/tea and maybe even a sandwich.
+
+## Running Cypress Accessibility Tests
+
+Once everything is installed, the a11y tests can be invoked by first making sure the server is running in docker, following the instructions for a normal build run, and then once the server is up and running, separately running the command:
+
+```
+npm run build:dev:a11y
+```
+
+This checks that all the Cypress dependencies are installed, and then kicks off a dev-rebuild specifically for a11y testing, and start up cypress locally (outside of docker). Once open, select the a11y tests, and Cypress should automatically start running them.
```

Comment:
<img width="395" alt="image" src="https://user-images.githubusercontent.com/2896608/74874398-c0a9e600-5315-11ea-80fd-0081058582cc.png">

Reply:
(still getting this after i ran `inv docker-catch-up`)

Reply:
try `inv docker-npm install`

Reply:
the docs forgot to mention that you also need to run `npm install` locally, not just in docker, to actually _run_ the tests